ocp_resources/resource.py

```diff
@@ -385,6 +387,15 @@ def __init__(
         self.teardown = teardown
         self.timeout = timeout
         self.delete_timeout = delete_timeout
+        self.node_selector = node_selector
+        self.node_selector_labels = node_selector_labels
+        self.body_node_selector = self._set_node_selector()
```

Comment:
Can you please rename to something which indicates that this applies to the spec, for example `self.spec_node_selector`